bundle/manifests/datasciencecluster.opendatahub.io_datascienceclusters.yaml

```diff
@@ -245,6 +245,53 @@ spec:
                         - Removed
                         pattern: ^(Managed|Unmanaged|Force|Removed)$
                         type: string
+                      nim:
+                        description: Configures and enables Nvidida's NIM integration
+                        properties:
+                          devFlags:
```

Comment:
do we need devFlags exclusively for nims? my expectation is that if we need these devFlags, they could go under the parent component (kserve)?

Reply:
seems to be a leftover, maybe a `make generate manifests api-docs` is required

Reply:
need `make generate manifests bundle`

Reply:
@etirelli @zdtsw missed that part, sorry about that. This has been updated.